lib/timeline/component/DataAxis.js

```diff
@@ -25,7 +25,7 @@ function DataAxis (body, options, svg, linegraphOptions) {
     labelOffsetX: 10,
     labelOffsetY: 2,
     iconWidth: 20,
-    width: parseInt(document.querySelector('.data-region.data-container-with-timeline').offsetWidth / 26) + 'px',
+    width: (document.querySelector('.data-region.data-container-with-timeline')) ? parseInt(document.querySelector('.data-region.data-container-with-timeline').offsetWidth / 26) + 'px' : '0px',
```

Comment:
Please not execute twice document.querySelector('.data-region.data-container-with-timeline')